README.md

```diff
@@ -1 +1,59 @@
-# tic-tac-toe-basic
\ No newline at end of file
+## Tic Tac Toe
+This Tic Tac Toe game is a command line application built in Ruby.
+The initial functionality permits gameplay between two human players. The program will exit when gameplay ends: either when a player has won, or if the game has reached a draw.
+
+## Table of Contents
+- [Getting Started](#getting_started)
+    - [Installation](#installation)
+    - [Linting](#linting)
+    - [Testing](#testing)
+- [Usage](#usage)
+    - [Launching the Game](#launching)
+    - [Instructions](#instructions)
+- [Future Implementation](#future)
+
+## Getting Started
+### Installation
+Clone this repo from the terminal:
+```
+git clone https://github.com/karen-olson/tic-tac-toe-basic
+```
+cd to the program directory & install dependencies:
+```
+bundle install
+```
+### Linting
+Run the linter:
+```
+bundle exec rubocop
+```
+To safely autocorrect correctable offenses:
+```
+bundle exec rubocop -a
+```
+### Testing
+Run tests:
+```
+bundle exec rspec
+```
+To automate the test suite:
+```
+bundle exec guard init rspec
```

Comment:
I am wondering if this is necessary? What does `init rspec` do? Is this something that users should run every time, or is this a one-time command needed to generate some boilerplate guard configuration?

Reply:
Good thought! I moved that command into the "installation" section.